addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxPlayerHandler.java

```diff
@@ -592,12 +591,9 @@ int currentRepeat() {
      * Ticks away when in a play state to keep current track time
      */
     private void timeCounter() {
-        Runnable runnable = new Runnable() {
-            @Override
-            public void run() {
-                if (playing) {
-                    updateChannel(mac, CHANNEL_CURRENT_PLAYING_TIME, new DecimalType(currentTime++));
-                }
+        Runnable runnable = () -> {
```

Comment:
You can write this in a single line instead of defining a variable. 
```
timeCounterJob = scheduler.scheduleWithFixedDelay(() -> {
    if (playing) {
        updateChannel(mac, CHANNEL_CURRENT_PLAYING_TIME, new DecimalType(currentTime++));
    }
}, 0, 1, TimeUnit.SECONDS);
```